pages/api/account/account.js

```diff
@@ -0,0 +1,81 @@
+import logger from "@config/logger";
+import { authOptions } from "../auth/[...nextauth]";
+import { getServerSession } from "next-auth/next";
+import connectMongo from "@config/mongo";
+import { Account } from "@models/index";
+
+export default async function handler(req, res) {
+  const session = await getServerSession(req, res, authOptions);
+
+  if (!session) {
+    res.status(401).json({ message: "You must be logged in." });
+    return;
```

Comment:
I like the style on line 16, where the return is on the same line as the `res` - I think we can use that style in other places of the code 👍 

```suggestion
    return res.status(401).json({ message: "You must be logged in." });
```

Reply:
Sure. But I think now you have merged the PR. Should I raise a new commit?

Reply:
no problem, I updated already, it was just info for next time